'make all' to generate bindings gives error but running generate-bindings.sh works.

<!-- Please search for open issues that relate to the same problem before opening a new one. -->

#### System information

- **OS & Version**: Linux/Ubuntu 20.04.2 LTS
- **Go version:**  go1.16.5 linux/amd64
- **razor-go version:** v1.0.0

#### Describe the bug
- Able to generate bindings by just running generate-bindings.sh , but aiming the same thing to be done by make all gives an error.

#### Expected behaviour
-When running make all ,its expected the bindings to be generated but there's an error in pipefail and Makefile.

#### Actual behaviour
ERROR :
Installing contract dependencies...
generate-bindings.sh: 3: set: Illegal option -o pipefail
make: *** [Makefile:11: fetch_bindings] Error 2

-Tried to fix this by removing the set -e -o pipefail line , there's still an Makefile error which looks quiet strange.

ERROR: 
Installing contract dependencies...

generate-bindings.sh: 16: Syntax error: "(" unexpected
make: *** [Makefile:11: fetch_bindings] Error 2



<!-- When submitting logs: please submit them as text and not screenshots. -->

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 GO ?= go
-SHELL = sh
+SHELL = /bin/bash
 BIN_DIR = ./build/bin
 RAZOR = ${BIN_DIR}/razor
 

--- a/generate-bindings.sh
+++ b/generate-bindings.sh
@@ -1,4 +1,4 @@
-#!/bin/zsh
+#!/bin/bash
 
 set -e -o pipefail
 

